libs/sdk-ui-tests-e2e/scripts/run_boiler_app.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# (C) 2021-2023 GoodData Corporation
+
+log() {
+  local now;
+  now=$(date '+%Y-%m-%d %H:%M:%S')
+  echo "$now $*" 1>&2
+}
+
+health_check() {
+    for ((i = 1; i <= 50; i++)); do
+        log "Check Boiler app is up, try $i"
+        if curl -k https://127.0.0.1:8080 ; then
+            return 0
+        else
+            sleep 3
+        fi
+    done
+    return 1
+}
+
+apt-get update && apt-get install -y curl && apt-get install -y jq && apt-get install -y sed && apt-get clean
+
+npx --yes @gooddata/$APP_VERSION init $APP_NAME --language $APP_LANGUAGE
+cd $APP_NAME
+echo " SDK_BACKEND=${SDK_BACKEND}"
+echo " TEST_WORKSPACE_ID=${TEST_WORKSPACE_ID}"
+echo " HOST=${HOST}"
+echo " TEST_BACKEND=${TEST_BACKEND}"
+
+jq --arg a "${TEST_BACKEND}" '.gooddata.hostname = $a' package.json > "tmp" && mv "tmp" package.json
+jq --arg a "${TEST_WORKSPACE_ID}" '.gooddata.workspaceId = $a' package.json > "tmp" && mv "tmp" package.json
+jq --arg a "${SDK_BACKEND}" '.gooddata.backend = $a' package.json > "tmp" && mv "tmp" package.json
+
+cat > .env <<-EOF
+TIGER_API_TOKEN=${TIGER_API_TOKEN}
```

Comment:
Is this conflicted with `.env` created above?

Reply:
no it's not, this env for boiler app only